pom.xml

```diff
@@ -178,7 +178,7 @@
         <dependency>
             <groupId>com.bencodez</groupId>
             <artifactId>votingplugin</artifactId>
-            <version>6.12.1</version>
+            <version>LATEST</version>
```

Comment:
`LATEST` shouldn't be used as a version and should be an actual version number instead.

Reply:
Fixed.